src/components/ProjectDescription/index.jsx

```diff
@@ -1,28 +1,25 @@
 import React from "react";
 import "./index.css";
-import gift from "./gift.jpg";
 import { Container, Row, Col } from "react-bootstrap";
+import { useTranslation } from "react-i18next";
+import gift from "./gift.jpg";
```

Comment:
Indicate that this an image, by adding img to the end so the reader would understand it

Reply:
PTAL